### PR TITLE
refactor(console): separate mdx usages from docs

### PR DESCRIPTION
**Pull request by demonzoo**
## Summary

Separate mdx usages from `docs`.

We decided not to copy mdx files from docs, as there are too many differences between the two packages, and re-using the mdx files at this point seems to be more cumbersome than convenient.

So separate the usages until we find a better solution.

### CODE REVIEW

File: packages/console/src/assets/docs/tutorial/integrate-sdk/android.mdx [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+		val logtoConfig = LogtoConfig(
+            endpoint = "<your-logto-endpoint>",
+            appId = "<your-app-id>",
+            scopes = null,
+            resources = null,
+            usingPersistStorage = true,
+        )
+
+		val logtoClient = LogtoClient(logtoConfig, application)
+}
```
Comment (IceHe) [inline packages/console/src/assets/docs/tutorial/integrate-sdk/android.mdx:94]: wrong ident

File: packages/console/src/assets/docs/tutorial/integrate-sdk/android.mdx [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+
+			LogtoConfig logtoConfig = new LogtoConfig(
+                "<your-logto-endpoint>",
+                "<your-app-id>",
+                null,
+                null,
+                true
+		    );
+	
+			logtoClient = new LogtoClient(logtoConfig, getApplication());
```
Comment (IceHe) [inline packages/console/src/assets/docs/tutorial/integrate-sdk/android.mdx:121]: ditto

File: packages/console/src/assets/docs/tutorial/integrate-sdk/android.mdx [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+logtoClient.getAccessToken { logtoException: LogtoException?, result: AccessToken? ->
+	// custom logic
+}
+
+logtoClient.getIdTokenClaims { logtoException: LogtoException?, result: IdTokenCliams? ->
+	// custom logic
+}
+
+logtoClient.fetchUserInfo { logtoException: LogtoException?, userInfoResponse: UserInfoResponse? ->
+		// custom logic
```
Comment (IceHe) [inline packages/console/src/assets/docs/tutorial/integrate-sdk/android.mdx:198]: ditto

File: packages/console/src/assets/docs/tutorial/integrate-sdk/react.mdx [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+
+```typescript
+import React from "react";
+import { useLogto } from '@logto/react';
+
+const App = () => {
+	const { isAuthenticated, signIn } = useLogto();
+
+  if !(isAuthenticated) {
+		return <SignInButton />
```
Comment (IceHe) [inline packages/console/src/assets/docs/tutorial/integrate-sdk/react.mdx:136]: ditto

File: packages/console/src/assets/docs/tutorial/integrate-sdk/react.mdx [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+```
+
+### Retrieve Auth Status
+
+```typescript
+import React from "react";
+import { useLogto } from '@logto/react';
+
+const App = () => {
+	const { isAuthenticated, signIn } = useLogto();
```
Comment (IceHe) [inline packages/console/src/assets/docs/tutorial/integrate-sdk/react.mdx:133]: ditto

**Review by IceHe — COMMENTED** [review]
Seems there are tabs and whitespaces in the `*.mdx`

File: packages/console/src/assets/docs/tutorial/integrate-sdk/android.mdx (hunk 1 above)
Comment (demonzoo) [reply packages/console/src/assets/docs/tutorial/integrate-sdk/android.mdx:94]: Android mdx is a direct copy from `docs`, maybe @xiaoyijun can help double check this?

File: packages/console/src/assets/docs/tutorial/integrate-sdk/android.mdx (hunk 3 above)
Comment (demonzoo) [reply packages/console/src/assets/docs/tutorial/integrate-sdk/android.mdx]: Fixed

**Review by IceHe — APPROVED** [review]
LGTM

**Pull request merged**